### ISSUE TITLE
5.0rc1 Unified Installer fails with static-lxml. 

### ISSUE THREAD

**Issue by liconti** [plone/Installers-UnifiedInstaller#41]
Running:
```
sudo ./install.sh zeocluster --target="/opt/plone/test5.0rc1" --clients=2 --build-python --static-lxml
```
results in the following error message:
```
Building lxml version 3.4.4.
Building without Cython.
Using build configuration of libxslt 1.1.17
Minimum required version of libxslt is 1.1.17, found 1.1.23
Building against libxml2/libxslt in the following directory: /usr/lib
error: Setup script exited with error: command 'gcc' failed with exit status 1
An error occurred when trying to install lxml 3.4.4. Look above this message for any errors that were output by easy_install.
While:
  Installing client1.
  Getting distribution for 'lxml==3.4.4'.
Error: Couldn't install: lxml 3.4.4
```
Changing lxml version in lxml_static.cfg to 3.4.4 fix the problem.

Thanks for your work

**Comment by smcmahon** [comment]
Thanks! We'll get that fixed in the next release.

**Comment by smcmahon** [comment]
Fixed in 5.0rc2 installer.
